src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const body = document.body;
+const logo = document.querySelector('.logo');
+const revolvedMessage = 'Promise was resolved!';
+const errorMessage = 'Promise was rejected!';
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+promise1.then(() => {
+  body.insertAdjacentHTML('beforeend', `<div class="message">
+    ${revolvedMessage}
+  </div>`);
```

Comment:
you may create common function for inserting this notification, and just call it, passing `success` or not